packages/http-response-serializer/index.js

```diff
@@ -43,7 +43,7 @@ const middleware = (opts, handler, next) => {
     if (!test) { return false }
 
     // if the response is null or undefined, normalizes it back to an object
-    handler.response = handler.response || {}
+    handler.response = handler.response != null ? handler.response : {}
```

Comment:
Shouldn't this be `!==` or am I missing some edge case here?

Reply:
@lmammino `!=` is safer as `handler.response !== null` will not catch cases where `handler.response` is `undefined`

Reply:
Shall we be a bit more explicit then and define this as follows?

```javascript
handler.response = (handler.response !== null && typeof handler.response !== 'undefined')  ? handler.response : {}
```

What do you think?

Reply:
I think it's pretty [common practice](https://stackoverflow.com/questions/2647867/how-to-determine-if-variable-is-undefined-or-null) and I find the alternative a little too verbose, but your call :)

Reply:
See here @lmammino  https://github.com/middyjs/middy/pull/477/commits/c15631bf58de417fb2f81de3125d918cfea181b3

Reply:
Awesome, thanks for your patience. I'll version bump and merge in the next few hours!